clients/include/blas3/testing_hemm.hpp

```diff
@@ -55,11 +55,12 @@ void testing_hemm_bad_arg(const Arguments& arg)
     hipblasSideMode_t side = HIPBLAS_SIDE_LEFT;
     hipblasFillMode_t uplo = HIPBLAS_FILL_MODE_LOWER;
 
-    int64_t colsA = side == HIPBLAS_SIDE_LEFT ? N : M;
+    size_t dim_A = (side == HIPBLAS_SIDE_LEFT ? N : M);
```

Comment:
Hmmm, think this flipped as discussed in rocBLAS.  Left would be M x M otherwise N x N.  @daineAMD  ?

Reply:
Yes, should be swapped.